tools/cockpit.spec

```diff
@@ -481,6 +482,8 @@ if [ "$1" = 1 ]; then
     mkdir -p /etc/motd.d /etc/issue.d
     ln -s ../../run/cockpit/motd /etc/motd.d/cockpit
     ln -s ../../run/cockpit/motd /etc/issue.d/cockpit.issue
+    printf "# List of users which are not allowed to login to Cockpit\nroot\n" > /etc/cockpit/disallowed-users
+    # chmod 600 /etc/cockpit/disallowed-users
```

Comment:
Hmm -- I have a feeling 644 is ok, but in the face of dynamic `umasks` I think it's a good idea to explicitly set permissions either way. 600 would be more cautious, also fine for me. Please pick one, and make it consistent everywhere (including the %ghost)